src/docs/sdk/event-payloads/properties/op.mdx

```diff
@@ -0,0 +1,10 @@
+`op`:
+
+: _Recommended_. Short code identifying the type of operation the span is measuring.
+If no set we fallback to `default` on the Server.
```

Comment:
```suggestion
If no op is set we fallback to `default` on the Server.
```

Reply:
This was marked as resolved without applying the change, assuming it was an accident.

TBH, I'd rather we give guidance on `op` values than saying what Relay does (I consider that an implementation detail). I wouldn't mind if we removed this sentence altogether.